### PR TITLE
Update item_name.csv

### PR DESCRIPTION
**Pull request by Oremal**
Add and Fix French Translation
0xeD40527E9FB58D436Be38418cffe0000c7574ea8 
### Description

1. Problems to solve.
2. Features to add.

### How to test

1. First step.

### CODE REVIEW

**Review by Namyujeong — CHANGES_REQUESTED** [review]
@Oremal Wrong conflict resolved. There are several same lines. Please check it again.

**Comment by Oremal** [comment]
@Namyujeong so, seems like when I switch to planetarium:development, all my line went *2, I think i cleaned it, please let me know if it's ok. Thanks Again.

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv [hunk 1]
```diff
@@ -338,9 +338,5 @@ ITEM_NAME_40500009,Gray Tail,회색 꼬리,Cauda Cinzento,,灰色の尻尾,Graue
 ITEM_NAME_40500010,Blue Ghost Tail,파란 유령 꼬리,Cauda Azul Fantasmagórica,,青ゴーストの尻尾,Blauer Geisterschwanz,,หางผีสีฟ้า,Niebieski ogon ducha,,,Ekor Hantu Biru,Queue Fantômatique Bleue,,កន្ទុយខ្មោចខៀវ,,蓝鬼尾巴,
 ITEM_NAME_49900001,Legendary Adventurer,전설의 모험가,Aventureiro Lendário,,伝説の冒険者,Legendärer Abenteurer,,นักผจญภัยในตำนาน,Legendarny poszukiwacz przygód,,,Petualang Legendaris,Aventurier Légendaire,,អ្នកផ្សងព្រេង Legendary,,传奇冒险家,
 ITEM_NAME_49900002,The First Warrior,최초의 전사,O primeiro guerreiro,,まず戦士,Der erste Krieger,El primer guerrero,นักรบแรก,Pierwszy Wojownik,Pirmasis karys,De eerste Warrior,Pertama Prajurit,Le premier guerrier,Первый воин,នេះជាអ្នកចម្បាំងដំបូង,,第一勇士,
-ITEM_NAME_49900003,Lucky Adventurer,행운의 모험가,Aventureiro da sorte,,ラッキー迷惑な冒険,Glücklicher Abenteurer,Aventurero afortunado,นักผจญภัยผู้โชคดี,Lucky Adventurer.,Lucky Adventurer.,Lucky Adventurer,Lucky Adventurer.,Aventurier chanceux,Lucky Adventurer,សំណាងអ្នកផ្សងព្រេង,,幸运的冒险家,
```
Comment (Namyujeong) [inline nekoyume/Assets/StreamingAssets/Localization/item_name.csv:341]: @Oremal Please don't erase the bottom three lines from this line.

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv [hunk 2]
```diff
@@ -338,9 +338,5 @@ ITEM_NAME_40500009,Gray Tail,회색 꼬리,Cauda Cinzento,,灰色の尻尾,Graue
...
-ITEM_NAME_49900004,Master Blacksmith,Master Blacksmith,Blacksmith principal.,,マスター鍛冶屋,Master Blacksmith.,Master Blacksmith,ช่างตีเหล็กโท,Master Blacksmith.,Master Blacksmith,Meester smid,Master Blacksmith.,Maître Blacksmith,Мастер Кузнец,Mastermmith,,掌握铁匠,
-ITEM_NAME_49900005,Sushi Frontier,스시 개척자,Fronteira de sushi.,,寿司フロンティア,Sushi Frontier.,Frontera de sushi,Sushi Frontier,Frontier Sushi.,Sushi pasienyje,Sushi frontier,Sushi Frontier.,Sushi frontière,Суши границы,ព្រំដែនស៊ូស៊ី,,寿司边疆,
 ITEM_NAME_500000,AP Potion,AP 포션,Poção AP,,APポーション,AP-Trank,,ยา AP,Mikstura AP,,,Ramuan AP,Potion de PA,,AP Potion,,AP药水,
 ITEM_NAME_600000,Chest,상자,Baú,,チェスト,Truhe,,กล่อง,Klatka piersiowa,,,Peti,Coffre,,ហិប,,胸部,
-ITEM_NAME_700000,Season medal of Arena 0,아레나 시즌 0 메달,Medalha de temporada de arena 0,,アリーナの季節メダル0.,Saisonmedaille der Arena 0,Medalla de temporada de arena 0,ฤดูกาลเหรียญของสนามกีฬา 0,Medal sezonu Areny 0,Sezono medalis Arena 0,Seizoen medaille van arena 0,Medali Musim Arena 0,Médaille de saison de l'arène 0,Сезонная медаль Арены 0,មេដាយមានមេដាយសង្វៀន 0,,竞技场的季节奖牌0,
```
Comment (Namyujeong) [inline nekoyume/Assets/StreamingAssets/Localization/item_name.csv:346]: @Oremal Please don't erase this line.

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv [hunk 3]
```diff
@@ -19,163 +19,163 @@ ITEM_NAME_201012,Great Steak,모둠스테이크,Bife Maravilhoso,,すごいス
 ITEM_NAME_201013,Legendary Steak,전설의 스테이크,Bife Lendário,,伝説のステーキ,Legendäres Steak,,สเต็กในตำนาน,Legendarny stek,,,Steak Legendaris,Steak Légendaire,,សាច់អាំង Legendary,,传奇牛排,
 ITEM_NAME_10100000,Wooden Club,나뭇가지,Taco de Madeira,,棍棒,Holzknüppel,,ไม้กระบอง,Drewniana pałka,,,Pentungan Kayu,Gourdin en Bois,,ក្លឹបឈើ,,木棍,
 ITEM_NAME_10110000,Long Sword,롱 소드,Espada Longa,,ロングソード,Langschwert,,ดาบยาว,Długi miecz,,,Pedang Panjang,Épée Longue,,ដាវវែង,,长剑,
-ITEM_NAME_10111000,Long Sword,롱 소드,Espada Longa,,ロングソード,Langschwert,,ดาบยาว,Długi miecz,,,Pedang Panjang,Épée Longue,,ដាវវែង,,长剑,
-ITEM_NAME_10112000,Long Sword,롱 소드,Espada Longa,,ロングソード,Langschwert,,ดาบยาว,Długi miecz,,,Pedang Panjang,Épée Longue,,ដាវវែង,,长剑,
-ITEM_NAME_10113000,Long Sword,롱 소드,Espada Longa,,ロングソード,Langschwert,,ดาบยาว,Długi miecz,,,Pedang Panjang,Épée Longue,,ដាវវែង,,长剑,
-ITEM_NAME_10114000,Long Sword,롱 소드,Espada Longa,,ロングソード,Langschwert,,ดาบยาว,Długi miecz,,,Pedang Panjang,Épée Longue,,ដាវវែង,,长剑,
+ITEM_NAME_10111000,Long Sword (Fire),롱 소드(불),Espada Longa(Fogo),,ロングソード（火）,Langschwert,,ดาบยาว (ไฟ),Długi miecz,,,Pedang Panjang,Épée Longue (Feu),,ដាវវែង,,长剑,
```
[the remark this answers is not in the document]
Comment (Oremal) [reply nekoyume/Assets/StreamingAssets/Localization/item_name.csv:22]: looks like i worked on the old version of the file, i'll bee correcting that

[the remark this answers is not in the document]
Comment (Oremal) [reply nekoyume/Assets/StreamingAssets/Localization/item_name.csv]: done

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv [hunk 4]
```diff
@@ -226,77 +226,77 @@ ITEM_NAME_305000,Ice Dye,얼음의 염료,Tintura de Gelo,,氷の染料,Eisfarbs
...
 ITEM_NAME_306031,Rigid Chain,단단한 쇠사슬,Cadeia rígida,,リジッドチェーン,Rigid Chain,cadena rígida,โซ่แข็ง,sztywny łańcuch,standus grandinės,Rigid Chain,Rantai kaku,chaîne rigide,Жесткая Цепная,ខ្សែសង្វាក់រឹង,,刚性链条,
 ITEM_NAME_306032,Broken Wand,화염의 부러진 지팡이,Varinha Quebrada,,壊れたワンド,Zerbrochener Zauberstab,,ไม้กายสิทธิ์หัก,Broken Wand,,,Tongkat Rusak,Baguette Cassée,,ដំបង​វេទមន្តខូច,,断魔杖,
-ITEM_NAME_306033,Saehrimnir’s Axe,화염의 제림니르 도끼,Machado de Saehrimnir,,セリムニルのアックス,Saehrimnirs Axt,,ขวานของ Saehrimnir,Topór Saehrimnira,,,Kapak Saehrimnir,Hache de Saehrimnir,,ពូថៅរបស់ Saehrimnir,,塞里米尔的斧头,
-ITEM_NAME_306034,Saehrimnir's Mace,화염의 제림니르 철퇴,Clava de Saehrimnir,,セリムニルのメイス,Saehrimnirs Keule,,กระบองของ Saehrimnir,Maczuga Saehrimnira,,,Tongkat Kebesaran Saehrimnir,Masse de Saehrimnir,,Mace របស់ Saehrimnir,,塞里米尔的钉头锤,
+ITEM_NAME_306033,Serimnir’s Axe,화염의 제림니르 도끼,Machado de Serimnir,,セリムニルのアックス,Serimnirs Axt,,ขวานของ Serimnir,Topór Serimnira,,,Kapak Serimnir,Hache de Serimnir,,ពូថៅរបស់ Serimnir,,塞里米尔的斧头,
```
[the remark this answers is not in the document]
Comment (Oremal) [reply nekoyume/Assets/StreamingAssets/Localization/item_name.csv]: done

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv (hunk 1 above)
Comment (Oremal) [reply nekoyume/Assets/StreamingAssets/Localization/item_name.csv:341]: done

File: nekoyume/Assets/StreamingAssets/Localization/item_name.csv (hunk 2 above)
Comment (Oremal) [reply nekoyume/Assets/StreamingAssets/Localization/item_name.csv]: done

**Review by Namyujeong — APPROVED** [review]

**Comment by Namyujeong** [comment]
@Oremal All check is done, Thanks for your first contribution!

**Pull request merged**